docs/articles/blog/2018-11-7-testcafe-v0-23-1-released.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: TestCafe v0.23.1 Released
+permalink: /blog/:title.html
+---
+# TestCafe v0.23.1 Released
+
+Select tests and fixtures to run by their metadata and run dynamically loaded tests.
```

Comment:
"by their metadata"? How do you run tests by their metadata?

Reply:
This file is the same as the previous one. The same questions/changes apply here.